### DISCUSSION TITLE
Multi GPU using OpenCL

### DISCUSSION THREAD

**Discussion by ccbadd** [LostRuins/koboldcpp#320] [Q&A] [answered]
The gpu options seem that you can select only one gpu when using OpenBLAST.  Is there any way to use dual gpus with OpenCL?  I have tried it with a single AMD card and two Intel ARC 770s but can't use both 770s.

**Comment by LostRuins** [comment, marked as answer]
Unfortunately, OpenCL does not support multple GPUs at this time.
